upgraderesponder/service.go

```diff
@@ -189,6 +193,91 @@ func (s *Server) createDB(name string) error {
+		"longhorn_setting_recurring_successful_jobs_history_limit":                         true,
+		"longhorn_setting_registry_secret":                                                 true,
+		"longhorn_setting_remove_snapshots_during_filesystem_trim":                         true,
+		"longhorn_setting_replica_auto_balance":                                            true,
+		"longhorn_setting_replica_file_sync_http_client_timeout":                           true,
+		"longhorn_setting_replica_replenishment_wait_interval":                             true,
+		"longhorn_setting_replica_soft_anti_affinity":                                      true,
+		"longhorn_setting_replica_zone_soft_anti_affinity":                                 true,
+		"longhorn_setting_restore_concurrent_limit":                                        true,
+		"longhorn_setting_restore_volume_recurring_jobs":                                   true,
+		"longhorn_setting_snapshot_data_integrity":                                         true,
+		"longhorn_setting_snapshot_data_integrity_cronjob":                                 true,
+		"longhorn_setting_snapshot_data_integrity_immediate_check_after_snapshot_creation": true,
+		"longhorn_setting_storage_minimal_available_percentage":                            true,
+		"longhorn_setting_storage_network":                                                 true,
+		"longhorn_setting_storage_over_provisioning_percentage":                            true,
+		"longhorn_setting_storage_reserved_percentage_for_default_disk":                    true,
+		"longhorn_setting_support_bundle_failed_history_limit":                             true,
+		"longhorn_setting_system_managed_components_node_selector":                         true,
+		"longhorn_setting_system_managed_pods_image_pull_policy":                           true,
+		"longhorn_setting_taint_toleration":                                                true,
+		"longhorn_volume_average_actual_size_gib":                                          true,
+		"longhorn_volume_average_number_of_replicas":                                       true,
+		"longhorn_volume_average_size_gib":                                                 true,
+		"longhorn_volume_average_snapshot_count":                                           true,
+	}
+}
+
+func (s *Server) isTagWithPrefix(tag string) bool {
+	includePrefixes := []string{
+		"longhorn_node_disk_",            // longhorn_node_disk_<hdd/ssd/nvme/unknown>_count
+		"longhorn_volume_access_mode_",   // longhorn_volume_access_mode_<rwo/rwx/unknown>_count
+		"longhorn_volume_data_locality_", // longhorn_volume_data_locality_<disabled/best_effort/strict_local/unknown>_count
+		"longhorn_volume_frontend_",      // longhorn_volume_frontend_<blockdev/iscsi/unknown>_count
+	}
+	for _, prefix := range includePrefixes {
+		if strings.HasPrefix(tag, prefix) {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
Sorry for the confusion. This repo is consumed by other projects to. So a white list tag should be configable via configmap similar to a resp.json file. I can take care of this one in https://github.com/longhorn/upgrade-responder/issues/47